widgets/panels.py

```
@@ -8,9 +8,11 @@
     QLineEdit,
     QScrollArea,
     QTextEdit,
+    QSizePolicy,
```

Ruff (0.11.9)

`PySide6.QtWidgets.QSizePolicy` imported but unused

Remove unused import: `PySide6.QtWidgets.QSizePolicy`

(F401)